netcheck.sh

```diff
@@ -1,31 +1,28 @@
 #!/bin/bash
 
-################################################################################
-##               Netcheck - Simple internet connection logging                ##
-##               https://github.com/TristanBrotherton/netcheck                ##
-##                                       -- Tristan Brotherton                ##
-################################################################################
+#########################################################################################
+##               Netcheck - Simple internet connection logging updated                 ##
+##                          for use on travel router GL.INET AR750S-EXT                ##
+##                                                                                     ##
+##               https://github.com/jmizer1112/GLI.INET_AR750S-EXT-netcheck            ##
+##               -- Jerry D Mizer                                                      ##
+##               originally forked from https://github.com/TristanBrotherton/netcheck  ##
+##                                       -- Tristan Brotherton                         ##
+#########################################################################################
 
 VAR_SCRIPTNAME=`basename "$0"`
-VAR_SCRIPTLOC="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-VAR_CONNECTED=true
-VAR_LOGFILE=log/connection.log
-VAR_SPEEDTEST_DISABLED=false
+VAR_SCRIPTLOC="/lib"
+VAR_CONNECTED=false
+VAR_LOGFILE=/mnt/sda1/netcheck.log
```

Comment:
The value of this variable should be forced to come from the command line since the router doesn't have much storage alone for logs. Granted the script shouldn't make a huge log, but if the state of the connection is constantly cycling it may inflate the log file size.